Classes/Private/Libxml2/Converters/In/InAttributeConverter.swift

```diff
@@ -4,8 +4,8 @@ import libxml2
 extension Libxml2.In {
     class AttributeConverter: SafeConverter {
 
-        typealias Attribute = HTML.Attribute
-        typealias StringAttribute = HTML.StringAttribute
+        typealias Attribute = Libxml2.Attribute
+        typealias StringAttribute = Libxml2.StringAttribute
```

Comment:
I'd prefer to avoid an Aztec-wide definition for this for now - for the time being I'd prefer to let each class define its own `typealiases`.

I'm open to reviewing this once we're closer to final, though.